src/synapse/azext_synapse/_help.py

```diff
@@ -0,0 +1,374 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+# pylint: disable=line-too-long, too-many-lines
+
+
+helps['synapse'] = """
+type: group
+short-summary: Manage and operate Synapse Workspace, Spark Pool, Sql Pool.
+"""
+
+helps['synapse workspace'] = """
+type: group
+short-summary: Manage Synapse Workspace.
+"""
+
+helps['synapse workspace create'] = """
+type: command
+short-summary: Create a Synapse workspace.
+examples:
+  - name: Create a Synapse workspace
+    text: |-
+        az synapse workspace create --name fromcli4 --resource-group rg \\
+          --account-url https://testadlsgen2.dfs.core.windows.net --file-system testfilesystem \\
+          --sql-admin-login-user cliuser1 --sql-admin-login-password Password123! --location "East US"
+"""
+
+helps['synapse workspace list'] = """
+type: command
+short-summary: List all Synapse workspaces under a subscription or under a specific resource group.
+examples:
+  - name: List all Synapse workspaces under a subscription
+    text: |-
+        az synapse workspace list
+  - name: List all Synapse workspaces under a specific resource group
+    text: |-
+        az synapse workspace list --resource-group rg
+"""
+
+helps['synapse workspace show'] = """
+type: command
+short-summary: Get a Synapse workspaces with workspace name.
+examples:
+  - name: Get a Synapse workspaces with workspace name.
+    text: |-
+        az synapse workspace show --name testsynapseworkspace --resource-group rg --name testsynapseworkspace
+"""
+
+helps['synapse workspace update'] = """
+type: command
+short-summary: Update a Synapse workspace.
+examples:
+  - name: Update a Synapse workspace
+    text: |-
+        az synapse workspace update --name fromcli4 --resource-group rg \\
+          --tags key1=value1
+"""
+
+helps['synapse workspace delete'] = """
+type: command
+short-summary: Delete a Synapse workspaces with workspace name.
+examples:
+  - name: Delete a Synapse workspaces with workspace name.
+    text: |-
+        az synapse workspace delete --name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse workspace wait'] = """

```

Comment:
Place the CLI in a waiting state until a condition of a workspace is met. you can refer to the help from other 'wait'.

Reply:
> Place the CLI in a waiting state until a condition of a workspace is met. you can refer to the help from other 'wait'.

fixed.